src/presentation/api/routers/exams.py

```
@@ -0,0 +1,167 @@
+
+    # Exclude None values so SQLModel will use the model defaults for non-optional fields
+    payload = data.model_dump(exclude_none=True, exclude={"id"})
+    exam = Examination(**payload)
+    session.add(exam)
+    session.commit()
+    session.refresh(exam)
+    return exam
+
+
+@router.get("/{exam_id}", response_model=ExaminationRead)
+def get_exam(exam_id: int, session: Session = Depends(get_session)) -> Examination:
```

Ruff (0.14.3)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)